backend/Wheelingful.API/Program.cs

```diff
@@ -22,4 +35,6 @@
 
 app.MapControllers();
 
+app.MapIdentityPartialApi<IdentityUser>();
```

Comment:
should you add UseAuthentication on line 33?

Reply:
Yes, you are right. In fact, it was here. I have deleted it by accident.